astro/src/content/docs/lifecycle/authenticate-users/oauth/scopes.mdx

```diff
@@ -0,0 +1,157 @@
+---
+title: OAuth Scopes
+description: Learn about OAuth scope policy configuration, managing custom scopes, and using scopes in an OAuth2 workflow.
+navcategory: login-methods
+section: lifecycle
+subcategory: authenticate users
+tertcategory: oauth
+---
+import AdvancedEditionBlurb from 'src/content/docs/_shared/_advanced-edition-blurb.astro';
+import ApplicationScopesSettings from 'src/content/docs/_shared/_application-scopes-settings.mdx';
+import Aside from 'src/components/Aside.astro';
+import Breadcrumb from 'src/components/Breadcrumb.astro';
+import InlineField from 'src/components/InlineField.astro';
+import InlineUIElement from 'src/components/InlineUIElement.astro';
+import OAuthScopeMessageLookup from 'src/content/docs/_shared/_oauth-scope-message-lookup.mdx';
+
+## Overview
+
+<Aside type="version">
+  Available since 1.50.0
+</Aside>
+
+OAuth scopes provide you with another dimension for applying access controls to applications. Scopes are requested by OAuth clients (applications) during authorization, and optionally consented to by the user providing the authorization. The consented scopes make their way into the access token granted to the client, and can be checked when the client uses the access token to access protected resources.
+
+In FusionAuth, applications can be designated as first-party or third-party. First-party applications are often synonymous with the authorization server from the user's perspective, and don't require consent for requested scopes. Third-party applications, by contrast, require consent by an authorizing user for the scopes they request.
```

Comment:
Not a compound sentence because the second clause does not contain a noun.

```suggestion
In FusionAuth, applications can be designated as first-party or third-party. First-party applications are often synonymous with the authorization server from the user's perspective and don't require consent for requested scopes. Third-party applications, by contrast, require consent by an authorizing user for the scopes they request.
```